### PR TITLE
Workflow

### PR DESCRIPTION
**Pull request by ohjenssen**

### CODE REVIEW

**Comment by ohjenssen** [comment]
Pull request for the course assignment in"workflow"

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
```
Comment (Fermain) [inline .eslintrc.json:1]: Please run a JSON formatter on this file to fix indentation

File: README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Workflow course assignment
+
+[![Automated E2E Testing](https://github.com/ohjenssen/workflow-ca/actions/workflows/e2e-test.yml/badge.svg?branch=workflow-automated-testing)](https://github.com/ohjenssen/workflow-ca/actions/workflows/e2e-test.yml)
+[![Automated Unit Testing](https://github.com/ohjenssen/workflow-ca/actions/workflows/unit-test.yml/badge.svg?branch=workflow-automated-testing)](https://github.com/ohjenssen/workflow-ca/actions/workflows/unit-test.yml)
+
```
Comment (Fermain) [inline README.md:5]: I would expect to see the various commands listed here too like `npm i`, `npm run build`, `npm run start`

File: src/js/api/auth/login.test.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+};
+
+describe("login", () => {
+    it("stores a token when provided with valid credentials", async () => {
+        global.fetch = jest.fn(() => fetchSuccess()); // Setting which of our mock functions is going to replace fetch() in this situation. We are setting success because we wnat a succesful response
+        const mockLoginResponse = await login(TEST_EMAIL, TEST_PASSWORD);
+
+        global.localStorage.setItem("token", TEST_ACCESSTOKEN);
+        expect(mockLoginResponse).toEqual(TEST_USER);
+        expect(TEST_ACCESSTOKEN).toEqual(global.localStorage.getItem("token"));
```
Comment (Fermain) [inline src/js/api/auth/login.test.js:36]: You could also `expect(fetch).toHaveBeenCalledTimes(1)` to harden this up

File: src/js/api/posts/create.test.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        statusText: "OK", 
+        json: () => Promise.resolve(validReturn)
+    })
+}
+
+describe("create", () => {
+    it("returns a valid item with a valid input", async () => {
+        global.fetch = jest.fn(() => fetchSuccess());
+        const mockCreate = await createPost(TEST_TITLE, TEST_BODY, TEST_TAGS, TEST_MEDIA);
+
```
Comment (Fermain) [inline src/js/api/posts/create.test.js:35]: `expect(fetch).toHaveBeenCalledWith(YOUR_TEST_OBJECT)`

**Review by Fermain — APPROVED** [review]
There are only minor issues in this submission and overall the requirements have been met. The testing approach shows thought and lateral thinking that result in useful coverage.

**Pull request merged**